pkg/quota/v1/evaluator/core/resource_claims.go

```diff
@@ -187,6 +192,24 @@ func (p *claimEvaluator) getVerifiedPodUsage(claim *resourceapi.ResourceClaim) c
 	if pod.Status.ExtendedResourceClaimStatus != nil && pod.Status.ExtendedResourceClaimStatus.ResourceClaimName != claim.Name {
 		return nil
 	}
+	ownedClaims := p.claimGetter(claim.Namespace, pod.UID)
```

Comment:
I think we can limit this check to when `pod.Status.ExtendedResourceClaimStatus == nil` ... if the pod status specifically indicates the resource claim we're admitting is the singular extended claim, that's sufficient to ensure only a single claim is getting exempted from quota